allvers/2006-03-01-GPL-2.0-from-gnu.org.txt.notes

```diff
@@ -0,0 +1,6 @@
+Date: 20060301
+Link: http://web.archive.org/web/20060301101606if_/http://www.gnu.org:80/licenses/gpl.txt
+Notes: This version has "Franklin St" instead of "Franklin Street" in both
+ preamble and appendix. This is older than 5.1 but before 6.
+ There is also a regression wrt. Library vs. lesser
```

Comment:
There was no regression. The text version on GNU.org only had "Lesser" instead of "Library" a month after this one. Before that it only had "Library", see 2005-05-08.

Reply:
This is a dupe from the previous gnu.org text. Removed in 9c1b6fb1de3115f41fc00d8d82d7816ee17887f6